TicTacToe/first.js

```diff
@@ -0,0 +1,111 @@
+let gameCounter = 1;
+let playerOne = 0;
+let playerTwo = 0;
```

Comment:
Já vi nomes mais explícitos como:
```suggestion
let playerOneScore = 0;
let playerTwoScore = 0;
```

Mas pronto, serve